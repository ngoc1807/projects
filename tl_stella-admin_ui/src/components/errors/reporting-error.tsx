import { FC } from "react";
import { Typography, Box } from "@mui/material";
import ErrorIcon from "@mui/icons-material/Error";

export const ReportingError: FC = () => {
  return (
    <>
      <Box display="flex" alignItems="center">
        <ErrorIcon fontSize="large" sx={{ marginRight: 3 }} />
        <Box>
          <Typography variant="h6">
            There was an error loading the reporting numbers.
          </Typography>
          <Typography>
            Please refresh the page and try again. If the issue persists, please
            contact support.
          </Typography>
        </Box>
      </Box>
    </>
  );
};
