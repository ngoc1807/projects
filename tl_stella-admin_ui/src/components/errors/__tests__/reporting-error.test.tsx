import { cleanup, render } from "@testing-library/react";
import { ReportingError } from "../reporting-error";

describe("reporting-error", () => {
  afterEach(() => {
    cleanup();
  });
  it("should render correctly", () => {
    const { container } = render(<ReportingError />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
