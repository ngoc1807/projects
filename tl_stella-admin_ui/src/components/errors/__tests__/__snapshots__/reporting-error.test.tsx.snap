// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reporting-error should render correctly 1`] = `
<div
  class="MuiBox-root css-70qvj9"
>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge css-18ois63-MuiSvgIcon-root"
    data-testid="ErrorIcon"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
    />
  </svg>
  <div
    class="MuiBox-root css-0"
  >
    <h6
      class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
    >
      There was an error loading the reporting numbers.
    </h6>
    <p
      class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      Please refresh the page and try again. If the issue persists, please contact support.
    </p>
  </div>
</div>
`;
